### DISCUSSION TITLE
Is the "shakey jittery chart" glitch already gone? :-)

### DISCUSSION THREAD

**Discussion by xrysf03** [binjr/binjr#157] [General]
Dear everyone (and the author in particular), 
in previous versions of BinJr within the last half a year, I kept spotting a funny "visual glitch" where... when selecting a time interval for a zoom-in, by clicking and draggin the mouse, the chart would start to shake along the X axis - as if toggling between two nearby quantised scale factors.
Just wanted to report that I don't see it anymore, in the 3.20.0 preview that I'm testing just yet. I'm not entirely sure though, because I haven't worked in this version for too long yet.

**Comment by fthevenet** [comment]
I'm aware of one issue that sound like it could be what you've been experimenting, where a vertical scroll bar keeps appearing and disappearing on the pane where the charts are drawn as you move the mouse of the charts. This in turn, displaces the axis for the charts and causes that jittery look.
This issue appeared somewhat suddenly, I suspect alongside an update to the graphic framework that I use.
 I agree it is quite annoying, but I haven't had the time nor courage to dive deep enough to get to the bottom of this one so far...

It is always possible that the issue might have been fixed the same way that it appeared, i.e. with an update to the latest version of that dependency, and I have to say I haven't seen it in a while. 
But without a proper analysis, I'm quite reluctant to claim it's gone for good, unfortunately  :-/

**Comment by fthevenet** [comment]
It's been a _very_ long time coming, but I'm hopeful this is finally addressed by 42de40b34c68a334d3f5ffc75fa9634b0131a697 :tada:
